cmd/argocd/commands/admin/app_test.go

```diff
@@ -132,8 +132,8 @@ func TestGetReconcileResults_Refresh(t *testing.T) {
 
 	require.NoError(t, err)
 
-	assert.Equal(t, health.HealthStatusMissing, result[0].Health)
-	assert.Equal(t, v1alpha1.SyncStatusCodeOutOfSync, result[0].Sync.Status)
+	assert.Equal(t, health.HealthStatusProgressing, result[0].Health)
+	assert.Equal(t, v1alpha1.SyncStatusCodeSynced, result[0].Sync.Status)
```

Comment:
This change was needed, because I've added a `namespace` to the test deployment. It does have no effect on the test goal.